[2019-10-17 01:23:41,008] {taskinstance.py:616} INFO - Dependencies all met for <TaskInstance: dag_cluster.Begin_execution 2018-01-01T00:00:00+00:00 [queued]>
[2019-10-17 01:23:41,021] {taskinstance.py:616} INFO - Dependencies all met for <TaskInstance: dag_cluster.Begin_execution 2018-01-01T00:00:00+00:00 [queued]>
[2019-10-17 01:23:41,022] {taskinstance.py:834} INFO - 
--------------------------------------------------------------------------------
[2019-10-17 01:23:41,022] {taskinstance.py:835} INFO - Starting attempt 1 of 1
[2019-10-17 01:23:41,023] {taskinstance.py:836} INFO - 
--------------------------------------------------------------------------------
[2019-10-17 01:23:41,071] {taskinstance.py:855} INFO - Executing <Task(DummyOperator): Begin_execution> on 2018-01-01T00:00:00+00:00
[2019-10-17 01:23:41,073] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'dag_cluster', 'Begin_execution', '2018-01-01T00:00:00+00:00', '--job_id', '2', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/cluster_dag.py', '--cfg_path', '/tmp/tmphlut6wk6']
[2019-10-17 01:23:44,132] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,129] {plugins_manager.py:148} ERROR - No module named 'operators'
[2019-10-17 01:23:44,133] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution Traceback (most recent call last):
[2019-10-17 01:23:44,134] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/lib/python3.7/site-packages/airflow/plugins_manager.py", line 142, in <module>
[2019-10-17 01:23:44,134] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     m = imp.load_source(namespace, filepath)
[2019-10-17 01:23:44,135] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/lib/python3.7/imp.py", line 171, in load_source
[2019-10-17 01:23:44,136] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     module = _load(spec)
[2019-10-17 01:23:44,137] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 696, in _load
[2019-10-17 01:23:44,138] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
[2019-10-17 01:23:44,139] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
[2019-10-17 01:23:44,139] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
[2019-10-17 01:23:44,140] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/airflow/plugins/__init__.py", line 3, in <module>
[2019-10-17 01:23:44,141] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     import operators
[2019-10-17 01:23:44,144] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution ModuleNotFoundError: No module named 'operators'
[2019-10-17 01:23:44,145] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,132] {plugins_manager.py:149} ERROR - Failed to import plugin /usr/local/airflow/plugins/__init__.py
[2019-10-17 01:23:44,175] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,170] {plugins_manager.py:148} ERROR - No module named 'botocore'
[2019-10-17 01:23:44,175] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution Traceback (most recent call last):
[2019-10-17 01:23:44,176] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/lib/python3.7/site-packages/airflow/plugins_manager.py", line 142, in <module>
[2019-10-17 01:23:44,177] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     m = imp.load_source(namespace, filepath)
[2019-10-17 01:23:44,177] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/lib/python3.7/imp.py", line 171, in load_source
[2019-10-17 01:23:44,178] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     module = _load(spec)
[2019-10-17 01:23:44,178] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 696, in _load
[2019-10-17 01:23:44,179] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
[2019-10-17 01:23:44,179] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
[2019-10-17 01:23:44,179] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
[2019-10-17 01:23:44,180] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/airflow/plugins/custom_operator.py", line 8, in <module>
[2019-10-17 01:23:44,180] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     from airflow.hooks.S3_hook import S3Hook
[2019-10-17 01:23:44,181] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/S3_hook.py", line 19, in <module>
[2019-10-17 01:23:44,181] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution     from botocore.exceptions import ClientError
[2019-10-17 01:23:44,181] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution ModuleNotFoundError: No module named 'botocore'
[2019-10-17 01:23:44,182] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,175] {plugins_manager.py:149} ERROR - Failed to import plugin /usr/local/airflow/plugins/custom_operator.py
[2019-10-17 01:23:44,197] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,197] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-10-17 01:23:44,742] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,741] {dagbag.py:90} INFO - Filling up the DagBag from /usr/local/airflow/dags/cluster_dag.py
[2019-10-17 01:23:44,823] {base_task_runner.py:115} INFO - Job 2: Subtask Begin_execution [2019-10-17 01:23:44,822] {cli.py:516} INFO - Running <TaskInstance: dag_cluster.Begin_execution 2018-01-01T00:00:00+00:00 [running]> on host 614b7d808020
[2019-10-17 01:23:45,956] {logging_mixin.py:95} INFO - [[34m2019-10-17 01:23:45,955[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 0[0m
